sesman/config.h

```diff
@@ -60,6 +60,7 @@
 #define SESMAN_CFG_SEC_USR_GROUP     "TerminalServerUsers"
 #define SESMAN_CFG_SEC_ADM_GROUP     "TerminalServerAdmins"
 #define SESMAN_CFG_SEC_ALWAYSGROUPCHECK "AlwaysGroupCheck"
+#define SESMAN_CFG_RESTRICT_CLIPBOARD "RestrictOutboundClipboard"
```

Comment:
I think `SESMAN_CFG_RESTRICT_CLIPBOARD` should include `OUTBOUND` because this might mislead clipboard is restricted in both directions. It is a little bit too long but fair enough.